packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -352,10 +353,22 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
       version: this.options.version,
       darkMode: this.options.darkMode,
       session,
-      connected$: connection.connected$,
-      chainId$: this.dappDefaultChainSubject,
+      connected: false,
```

Comment:
`connection.connect();` is called several lines below, so i guess this value is not unreasonable.